docs/ad/cli.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: Anomaly Detection CLI
+parent: Anomaly Detection
+nav_order: 3
+---
+
+# Anomaly Detection CLI
+
+Anomaly detection CLI lets you implement anomaly detection APIs with the `esad` command.
+
+You can use the CLI to:
+
+* Create detectors
+* Start, stop, and delete detectors
+* Create named profiles to connect to your cluster
+
+Install the anomaly detection plugin to your Elasticsearch instance, run the CLI using MacOS or Linux, and connect to any valid Elasticsearch end-point.
+
+## Install
+
+Launch your local Elasticsearch instance and make sure you have the anomaly detection plugin installed.
+
+To install the anomaly detection CLI:
+
+1. Download and extract `esad` binaries from:
+```
+https://github.com/opendistro-for-elasticsearch/anomaly-detection/actions/runs/224422434
```

Comment:
I'd just make this a link. "Download and extract the [esad binaries](link-end)."

Do we really not have a better DL link than that? What about our S3 bucket?

Reply:
I'll check for the link.